ui/app/components/clients/history.js

```diff
@@ -85,8 +86,8 @@ export default class History extends Component {
 
   get isDateRange() {
     return !isSameMonth(
-      new Date(this.getActivityResponse.startTime),
-      new Date(this.getActivityResponse.endTime)
+      parseAPITimestamp(this.getActivityResponse.startTime),
```

Comment:
does this function become irrelevant if the backend makes any of its "tasked" changes? If so maybe make a comment to revisit if so and so it done. That way if someone else does the work in the future the old pattern can easily be tracked down. Ignore me if this will always be here regardless of future backend changes.

Reply:
I was planning to make a ticket on our end, that's linked to the backend, to revisit warning banners if/when those backend changes happen. I'll mention this part of the code!